tests/xsc_test.go

```diff
@@ -25,29 +15,41 @@ func initXscTest(t *testing.T, minXscVersion string, minXrayVersion string) {
 
 // This func validates minimal Xsc version.
 // Since Xsc was migrated into Xray from version 3.107.13, we need to check minimal Xray version from this version forward instead of Xsc version.
-// For features that are available before the migration we pass minXscVersion to check. If the utilized Xray version >= 3.107.13, the returned Xsc version will always suffice the check.
+// For features that are available before the migration we pass minXscVersion that will be checked if Xray version < 3.107.13, and also pass version 3.107.13 as minXrayVersion.
+// (If the utilized Xray version >= 3.107.13, the returned Xsc version will always suffice and will not be checked).
 // For features that were introduced only after the migration we pass only minXrayVersion to check and can leave minXscVersion blank.
 // In general minXscVersion should be provided only for features that were introduced before Xsc migration to Xray
 func validateXscAndXrayVersion(t *testing.T, minXscVersion string, minXrayVersion string) {
-	// Validate active Xsc server
-	currentXscVersion, err := GetXscDetails().GetVersion()
+	// We first validate our Xray version so we will not address the old Xsc endpoints if Xray version >= 3.107.13. This will lead to a failure and skip the test
+	currentXrayVersion, err := GetXrayDetails().GetVersion()
 	if err != nil {
 		t.Skip(err)
 	}
 
-	if minXscVersion != "" {
-		if err = clientUtils.ValidateMinimumVersion(clientUtils.Xsc, currentXscVersion, minXscVersion); err != nil {
+	afterMigration := true
+	if err = clientUtils.ValidateMinimumVersion(clientUtils.Xray, currentXrayVersion, utils.MinXrayVersionXscTransitionToXray); err != nil {
+		afterMigration = false
+	}
+
+	if afterMigration && minXrayVersion != "" {
```

Comment:
```suggestion
	if minXrayVersion != "" {
```
why are we validating min xray version only if after migration?